app/forms/waste_carriers_engine/declaration_form.rb

```diff
@@ -8,9 +8,11 @@ def self.can_navigate_flexibly?
       false
     end
 
+    validates :declaration, inclusion: { in: [1] }
+
     def initialize(transient_registration)
       super
-      self.declaration = @transient_registration.declaration
+      self.declaration = transient_registration.declaration
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.declaration = transient_registration.declaration
```